OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -676,6 +676,17 @@ static void HideChildWidget(Widget parent, string widgetId)
 			if (widget != null)
 				widget.IsVisible = () => false;
 		}
+
+		public static string SanitizePlayerName(Session.Client client)
+		{
+			if (Game.Settings.Game.SanitizeMutedPlayerNames && TextNotificationsManager.MutedPlayers.ContainsKey(client.Index))
```

Comment:
MutedPlayers is a dictionary, when the value is true only then the player is muted